Makefile

```diff
@@ -107,7 +107,7 @@ installer-pkg: $(STAGING_DIR)
 	@codesign $(CODESIGN_OPTS) --prefix=com.apple.container. --entitlements=signing/container-network-vmnet.entitlements $(join $(STAGING_DIR), libexec/container/plugins/container-network-vmnet/bin/container-network-vmnet)
 
 	@echo Creating application installer
-	@pkgbuild --root $(STAGING_DIR) --identifier com.apple.container-installer --install-location /usr/local $(PKG_PATH)
+	@pkgbuild --root $(STAGING_DIR) --identifier com.apple.container-installer --install-location /usr/local --version ${RELEASE_VERSION} $(PKG_PATH)
```

Comment:
This will break for any build that is not on a tag, due to the trailing text from `get describe --tags`.

This change will, if there is trailing text, grab the number of commits past the most recent tag and apply it as a fourth component so that you'll get something like:

```console
% pkgutil --pkg-info com.apple.container-installer
package-id: com.apple.container-installer
version: 0.1.0.52
volume: /
location: usr/local
install-time: 1750473741
```

If there's no trailing text, you'll just get the three-component tag.

Reply:
This is what I see when I run it locally:

```
kathrynbaldauf@Kathryns-MacBook-Pro container % pkgutil --pkg-info com.apple.container-installer
package-id: com.apple.container-installer
version: 0.1.0-53-g1cbb941
volume: /
location: usr/local
install-time: 1751054654
```

I think it's reasonable to have the short commit sha at the end for packages that are NOT official release packages, such as for a dev testing locally.

Reply:
Yep! My mistake, if pkgutil seems fine with these, I do like that we see everything from `git describe`.